pages/utils/sum.ts

```diff
@@ -0,0 +1,8 @@
+export function sum(a: number, b: number, format: "html" | "text" = "html") {
+  const result = a + b;
+  if (format === "html") {
+    return `<span>${result}</span>`;
+  } else {
+    return result;
```

Comment:
Would it be much more readable to implement this code block as shown below?

if(format === "text") return result:

return `<span>${result}</span>`

Reply:
How about we remove the `else` statement in this case?